sickbeard/subtitles.py

```diff
@@ -849,6 +849,10 @@ def dhm(td):
                              ep_to_sub['show_name'], ep_num)
                 continue
 
+            if sickbeard.SUBTITLES_STOP_AT_FIRST and ep_to_sub['subtitles']:
```

Comment:
just to explain: if setting is enabled and database have subs, we should stop searching for more.